modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -179,7 +179,7 @@ For a successful horizontal autoscaling of your Mule apps, review the following
 
 * Scale up and scale down performance can vary based on the replica size and the application type.
 
-* The policy is optimized for replica sizes with <850m. Larger replica sizes might take longer to scale.
+* The policy is optimized for replica sizes with <850m. Larger replica sizes might take longer to scale. HPA is only recommended for smaller CPU applications with average usage of 0.2vCPU.
```

Comment:
```suggestion
* The policy is optimized for replica sizes with <850m. Larger replica sizes might take longer to scale. HPA is recommended only for smaller CPU applications with average usage of 0.2vCPU.
```

Reply:
Per this rule: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=%22only%22%20(%22simply%2C%22%20%22just%22)